mlflow/tracking/_model_registry/utils.py

```diff
@@ -157,7 +157,7 @@ def _get_databricks_uc_rest_store(store_uri, **_):
     from mlflow.version import VERSION
 
     raise MlflowException(
-        f"Detected Unity Catalog model registry URI '{mlflow.get_registry_uri()}'. "
+        f"Detected Unity Catalog model registry URI '{store_uri}'. "
```

Comment:
Minor change: usually, the `store_uri` argument and `mlflow.get_registry_uri()` are the same thing. However, in some cases they can differ. For example, if a customer specifies a models:/ URI like `models:/profile@databricks-uc/mymodel/12` while the registry URI is set to `./local-file`, we expect to load the model from the UC registry, and would instantiate a UC registry store here